rollbar/__init__.py

```diff
@@ -1056,10 +1056,12 @@ def _build_server_data():
     # server environment
     server_data = {
         'host': socket.gethostname(),
-        'argv': sys.argv,
         'pid': os.getpid()
     }
 
+    # argv does not always exist in embedded python environments
+    server_data['argv'] = getattr(sys, 'argv', None)
```

Comment:
Instead of saving `None` in this case, can you check if `argv` exists and only add it to `server_data` when it does?